docs-site/content/ja/worktree/_index.md

```
@@ -82,43 +78,41 @@ flowchart TD
-# 開発開始
-$ claude
+# 進入したセッションからそのまま開発を開始
 > /moai run SPEC-AUTH-001
 > /moai sync SPEC-AUTH-001
 ```
 
+現在のセッションを保ったままワークツリーをもう 1 つ開きたいときは `--spawn` を付けます。
+tmux の新しいウィンドウで起動し、元のウィンドウはそのまま残ります:
+
+```bash
+$ moai glm -w SPEC-AUTH-002 --spawn
```

markdownlint-cli2 (0.23.1)

[warning] Dollar signs used before commands without showing output

(MD014, commands-show-output)